WordPress/WordPressTest/Comments/Controllers/FullScreenCommentReplyViewControllerTests.swift

```diff
@@ -0,0 +1,142 @@
+
+import XCTest
+@testable import WordPress
+
+class FullScreenCommentReplyViewControllerTests: XCTestCase {
+    private var controller: FullScreenCommentReplyViewController?
+    private var window: UIWindow?
+
+    override func setUp() {
+        self.controller = FullScreenCommentReplyViewController.newEdit()
```

Comment:
We can remove all the `self` here. Also, you can make `controller` and `window` not optional (`!`).